src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -127,6 +127,7 @@ export class AKSClusterSettingsComponent
   readonly MIN_COUNT_DEFAULT_VALUE = 1;
   readonly MAX_COUNT_DEFAULT_VALUE = 5;
   readonly DEFAULT_LOCATION = 'eastus';
+  readonly DEFAULT_VMSIZE = 'test';
```

Comment:
Is this valid option i have confirmed with @imharshita 

```suggestion
default value for aks vmsize both for cluster and md
```
![image](https://user-images.githubusercontent.com/17727069/192703675-239ea1c8-99ef-4335-8663-cefb7ac035da.png)